src/resources/views/crud/columns/inc/wrapper_start.blade.php

```diff
@@ -2,15 +2,21 @@
     // this is made available by columns like select and select_multiple
     $related_key = $related_key ?? null;
 
+    // define the wrapper element
+    $wrapperElement = isset($column['wrapper']['element']) ? $column['wrapper']['element'] : 'a';
+    $wrapperElement = !is_string($wrapperElement) && is_callable($wrapperElement)
+        ? $wrapperElement($crud, $column, $entry, $related_key)
+        : $wrapperElement ?? 'a';
+
     // each wrapper attribute can be a callback or a string
     // for those that are callbacks, run the callbacks to get the final string to use
     foreach($column['wrapper'] as $attribute => $value) {
         $column['wrapper'][$attribute] = !is_string($value) && is_callable($value) ? $value($crud, $column, $entry, $related_key) : $value ?? '';
```

Comment:
@pxpm quick question, `!is_string($value) && is_callable($value)` why to do this double check? If it's a callable it is not a string, right?

Reply:
Imagine that you create an element `<file>something</file>`, then in your wrapper to create the element you would do:

```php
'wrapper' => [
    'element' => 'file'
];
```

This is not a closure, but if you ask `is_callable('file')` it would return `true`, because `file` is a php function so it's callable.

Reply:
Got it @pxpm 👌 didn't know callable would return true with function names as strings.